AutomatedTesting/Gem/PythonTests/editor_test_testing/EditorTest_That_Crashes.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) Contributors to the Open 3D Engine Project
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+def EditorTest_That_Crashes():
+    import azlmbr.legacy.general as general
+    general.crash()
```

Comment:
How does this return a test failure? Does the function itself raise an assert or are editor crashes raising asserts themselves?

Reply:
The application will just crash, and that will be picked up by the editor_test utils by checking the return code

Reply:
Note that Asserts != Crashes.
An assert may not necessarily crash the editor depending on the sys_asserts cvar.
genera.crash() forcefully crashes the editor which is what we want.

If you check the code it just tries to write an invalid memory location